### ISSUE TITLE
Array Formulas

### ISSUE THREAD

**Issue by eurosoll** [qax-os/excelize#1529]
Hi

Does excelise support array formulas like hstack / vstack.

I tried to use hstack but when excel opens the files it prepends a @ to the formule (=@HSTACK(....))

Thanks for all your help.

**Comment by xuri** [comment]
Thanks for your issue. I don't sure if you are setting or getting HSTACK and VSTACK formula functions. The CalcCellValue of this library still working in progress, so currently, it doesn't support evaluating these functions, also reference the issue #65. But if you want to set the cell formula, Because the formula function HSTACK and VSTACK was introduced in the new version of the Excel application, so please add the _xlfn. prefix for these formula functions. Also reference the issue #1506. For example:

```go
f.SetCellFormula("Sheet1", "A1", "_xlfn.HSTACK(A2:C3,E2:G3)")
```

I've closed this issue, and merge this into the #65. If you have any questions, please let me know to reopen this anytime.

**Comment by eurosoll** [comment]
Thank you very much. That works.
